src/components/avatar/Avatar.js

```diff
@@ -55,21 +61,28 @@ class Avatar extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const { avatarUrl, className, name, id, size = '' }: Props = this.props;
+        const { avatarUrl, className, name, id, size = '', isPending = false }: Props = this.props;
```

Comment:
Will followup for the size one since the `= ''` here is not the default prop value, it's just used for the object lookup. We can add a "medium" value for that and make it the default.